shared/js/ui/templates/hamburger-menu.es6.js

```diff
@@ -8,7 +8,7 @@ module.exports = function () {
             More Options
         </h2>
         <nav class="pull-right hamburger-menu__close-container">
-            <a href="javascript:void(0)" class="icon icon__close js-hamburger-menu-close"></a>
+            <a href="javascript:void(0)" class="icon icon__close js-hamburger-menu-close" role="button" aria-label="Close options"></a>
```

Comment:
I think I accidentally covered some of your work in the other PR, sorry :/

https://github.com/duckduckgo/duckduckgo-privacy-extension/pull/275/files#diff-8aee66435835fb83ef8142641ead94daR11